### PR TITLE
Cedexis Radar

### PR DESCRIPTION
**Pull request by kendrick-k**
add of Cedexis Radar : http://www.cedexis.com/radar/ read http://www.cedexis.com/ (radar, openmix & impact)

### CODE REVIEW

File: libraries/cedexis/radar/v1/Tag.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+		async: true,
+		description: "Radar measure and collects Page Load Times, from around the globe. Used by Cedexis for DNS dynamic routing.",
+		html: "",
+		locationDetail: "",
+		isPrivate: false,
+		url: "",
+		usesDocWrite: false,
+		upgradeable: true,
+		parameters: [{
+			name: "Cedexis custommer ID",
```
Comment (peterfronc) [inline libraries/cedexis/radar/v1/Tag.js]: Spelling.

File: libraries/cedexis/radar/v1/Tag.js [hunk 2]
```diff
@@ -0,0 +1,49 @@
+//:include tagsdk-current.js
+
+qubit.opentag.LibraryTag.define("cedexis.radar.v1.Tag", {
+	config: {
+		/*DATA*/
+		name: "radar",
+		async: true,
+		description: "Radar measure and collects Page Load Times, from around the globe. Used by Cedexis for DNS dynamic routing.",
```
Comment (hllwd) [inline libraries/cedexis/radar/v1/Tag.js]: typo here too

**Comment by kendrick-k** [comment]
@peterfronc @hllwd I finally asked the vendor @faichelbaum to setup the tag : https://github.com/QubitProducts/opentag-library-sdk/pull/64 is it ok ?

**Comment by peterfronc** [comment]
Yes, sure, is it ready for review?

**Comment by faichelbaum** [comment]
Hi all

@peterfronc: yes you can

@hllwd: sadly, I can't as the dev who provided this specific version didn't pushed the uncompressed version on our github yet

**Comment by peterfronc** [comment]
We prefer to have it uncompressed for security reasons,  Ill need to check it in details, do you have the reference? Im sure its fine just wanna know the source.

**Comment by kendrick-k** [comment]
@faichelbaum works @ Cedexis http://www.cedexis.com/blog/author/francois/

**Pull request merged**